system/iptables_raw.py

```diff
@@ -4,6 +4,7 @@
 """
 (c) 2016, Strahinja Kustudic <strahinjak@nordeus.com>
 (c) 2016, Damir Markovic <damir@damirda.com>
+(c) 2016, Allen Sanabria <asanabria@linuxdynasty.org>
```

Comment:
In the Ansible community it is customary that if a user contributes to a module in any way. They will be added to the authors list. I do respect your decision though.